migrations/transformWordpressToUrlSlug.js

```diff
@@ -0,0 +1,29 @@
+// Copy the wordpress URL field into the urlSlug field for all blog posts
+// that have a wordpress URL set, and no Url Slug set.
+// Should be run with the contentful migration tool: https://github.com/contentful/contentful-migration
+
+module.exports = function (migration) {
+  migration.transformEntries({
+    contentType: 'post',
+    from: ['wordpress_url', 'urlSlug'],
```

Comment:
@irwalker See Slack#marketing_website—maybe I missed your description in my review…but I'm 100% certain we have things with a `wordpress_url` and this should be migrated.

Example:
https://app.contentful.com/spaces/91sm3pewxzag/entries/post_14